addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -626,5 +626,12 @@
           action="mrp_workcenter_productivity_report"
           groups="group_mrp_routings"
           sequence="12"/>
+
+    <menuitem id="menu_productivity_loss_action"
```

Comment:
Doesn't seems correct, it is a technical model. we shouldn't add record on it (instead of ones in the `mrp_data.xml`). It shouldn't be possible to remove the `category_productive` data record. Maybe juste re-add it fix the issue.

Also, note that the error message is very confusing and the model seems useless in master (it contains only a simple selection fields), I will create a task for it.

Reply:
I already added it through SA and it fixed the issue, yes, but it would make sense if the menu is accessible to the user with restrictions on mandatory ones.

Reply:
Ok I understand your point but even with the menu, we cannot create a Productivity Losses with a loss type of `performance` or `productive`. Also we shouldn't add menu in stable version. I create a task to clean this mess in master (2571416).